### DISCUSSION TITLE
MESH REMOTE DESKTOP AND WINDOWS 11 CLIENTS

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6141] [Issues]
Hi, I just upgraded on my mesh windows laptop to windows 11, and I am having problems controlling the remote desktop.  I was not able to move any windows, and they do not appear when I start them up.  Anyone else having this issue in Windows 11 Pro?  I appreciate any feedback.  Thx!

**Comment by gazetchik** [comment]
> Anyone else having this issue in Windows 11 Pro? 

Hey, man.
were you able to solve your issue with windows 11 pro?
